tests/unit/requirements.txt

```diff
@@ -1,5 +1,5 @@
-charmhelpers
+git+https://github.com/juju/charm-helpers.git#egg=charmhelpers
```

Comment:
This applies to requirements.txt and here.

Is there a reason that we're moving away from using the pypi released version of charmhelpers and instead using the trunk of the git repo instead?